tests/drivers/entropy/api/src/main.c

```diff
@@ -25,6 +25,11 @@
 #define BUFFER_LENGTH           10
 #define RECHECK_RANDOM_ENTROPY  0x10
 
+#ifdef CONFIG_RANDOM_BUFFER_NOCACHED
+__attribute__((__section__(".nocache")))
+static uint8_t buffer[BUFFER_LENGTH] = {0};
```

Comment:
Just a suggestion: move the definition of `buffer` here for targets where `CONFIG_RANDOM_BUFFER_NOCACHED` won't be selected, to improve readability. Something like this:

```
#ifdef CONFIG_RANDOM_BUFFER_NOCACHED
__attribute__((__section__(".nocache")))
static uint8_t buffer[BUFFER_LENGTH] = {0};
#else
static uint8_t buffer[BUFFER_LENGTH] = {0};
#endif
```

Reply:
done